### PR TITLE
feat: initial docs

### PR DESCRIPTION
**Pull request by GeorgeMac**
This is very much non-exhaustive, but I wanted to get what I have in while I switch to do the controller -> phase renaming.
I almost certainly need to run a spellcheck over it all too.

This adds a docsify page which can be rendered easily via GH pages.
It is all Markdown driven and I think materialized late via the JS snippest delivered via the CDN.
Should get us far enough I think.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+```
+
+The `glu.System` instance has methods for building new pipelines and configuring triggers.
+
+By delegating control to a `glu.System` instance we get a few out-of-the-box conveniences:
+
+1. Pre-built configuration format and parsing for accessing and authenticating the built-in sources (Git, OCI and so on).
+1. A CLI interface for inspecting and manually promoting resources on the command-line.
+1. An API interface for inspecting and manually promoting resources over a network.
+1. Lifecycle control (signal handling and graceful shutdown).
```
Comment (markphelps) [inline docs/README.md:68]: Should we add a bit about the ui you get ootb for visualizing your pipelines ?

File: README.md [hunk 2]
```diff
@@ -35,8 +36,14 @@ npm start
...
 In the future we plan to support more use-case, such as:
 
-- (In Progress) Progressive delivery (think Kargo / Argo Rollouts)
+- New sources:
+  - Helm
+  - Webhook / API
+  - Kubernetes (direct to cluster)
+- Progressive delivery (think Kargo / Argo Rollouts)
+  - Ability to guard promotion with condition checks on resource status
+  - Expose status via Go function definitions on resource types
```
Comment (markphelps) [inline README.md:49]: Should we also add a bit about the ability to view history as well as quickly revert to previous states?

**Review by markphelps — COMMENTED** [review]
Looks great. Just a couple suggestions

File: README.md (hunk 2 above)
Comment (GeorgeMac) [reply README.md:49]: Good shout!

File: docs/README.md (hunk 1 above)
Comment (GeorgeMac) [reply docs/README.md:68]: Good shout!

**Review by markphelps — APPROVED** [review]
Muy bueno

**Pull request merged**